app/javascript/packs/static.js

```diff
@@ -1 +1,11 @@
-import 'analytics';
+import "analytics";
+
+(function() {
+  // After the javascript has loaded request the fonts
+  let ss = document.createElement("link");
+  ss.type = "text/css";
+  ss.rel = "stylesheet";
+  ss.href =
+    "https://fonts.googleapis.com/css?family=Muli:300,400,700|Poppins:300,500";
```

Comment:
May want to run this by the security team, my feeling is google fonts is a sneaky way for them to add tracking software to projects 😛

Reply:
Fixed this! :tada:
https://github.com/brave-intl/publishers/pull/1854/commits/c7b0953757d4347c6b23beb60bc5201469f95ec7